Maple2.Server.Game/PacketHandlers/InstrumentHandler.cs

```diff
@@ -172,9 +172,58 @@ private enum Command : byte {
         session.Instrument = null;
     }
 
-    private void HandleJoinEnsemble(GameSession session, IByteReader packet) { }
+    private void HandleJoinEnsemble(GameSession session, IByteReader packet) {
```

Comment:
I have no idea how ensembles work anymore, but it does seem a bit strange to be that `JoinEnsemble` affects everyone in your party, but `LeaveEnsemble` only affects yourself. Is that correct?

Reply:
Yeah, it's correct. I think it might change a character's sync flags. If the packet was broadcasted to the field, it stops everyone from playing.